rdd/timestamp-receipts.md

```diff
@@ -0,0 +1,89 @@
+# Timestamp Receipts
+
+Timestamp Receipts are used to verify that a given file existed at a point in time by checking the hash of the timestamped file against a blockchain block.
+
+## Related Issues 
+- https://github.com/poetapp/node/issues/178
+- https://github.com/poetapp/node/issues/112
+- https://github.com/poetapp/node/issues/293
+
+## Receipt Generation
+
+Receipts are actual files a user requests, downloads and stores. These receipts contain all the data necessary to cryptographically prove that a file existed at a point in time.
+
+This is achieved by storing all the data necessary to regenerate the hash that was stored in the blockchain: the file that we want to validate and the hashes of other necessary files of the same batch.
+
+In practice, the verification algorithm requires: 
+- the hash of the block at which the timestamp was stored,
+- the verifiable claim that we want to validate, 
```

Comment:
If I am understanding correctly, which version of the claim it will need to be depends on the implementation. Implementation 1 would require a signed verifiable claim in order to generate the same hash I think? So its probably fine to leave this as verifiable claim as it could be either.

Reply:
or I also like your comment above that puts signed into brackets, `[signed] verifiable claim` but that might get too redudent using that wording everywhere in this document.

Reply:
> If I am understanding correctly, which version of the claim it will need to be depends on the implementation. Implementation 1 would require a signed verifiable claim in order to generate the same hash I think? So its probably fine to leave this as verifiable claim as it could be either.

Now that you mention it, I think you are right... but I also think we would need it to be signed in both implementations. 

Changing it to _signed_ for now, we can come back to this after we learn more during implementation.